HTTPService/pkg/entities/user.go

```diff
@@ -0,0 +1,12 @@
+package entities
+
```

Comment:
I have the doub, if we must have user entiti by service (http and grpc) or the same entiti  can be shared. it's to validate with seba or josh. (can you share me the response)